swarm/metrics/flags.go

```diff
@@ -94,5 +100,12 @@ func Setup(ctx *cli.Context) {
 				"host": hosttag,
 			})
 		}
+
+		if enableAccountingExport {
+			log.Info("Enabling accounting metrics export to InfluxDB")
```

Comment:
this block actually sends the data to influxdb, so the log line should say something more like "exporting accounting metrics to influxdb" rather than "enabling" them. it could also be switched to `log.Trace`. I don't see this necessary in `Info` level

Reply:
I don't have strong feelings about the level of this, but it is 1 line, so sometimes it is helpful to see it, even if you are running in INFO mode.

Reply:
👍 @JerzyLa please just change the text of the log-line